Tweet-311-Denver/tweet_311_denver_ui
# Textbox error gi

## What does this PR do?
- Adds `Welcome` page component
- Fixes error with text box click-in / out in `Tweet`
- Changes some widths to be responsive
- Add navigation to welcome page
-  Changes some styles to bold text and condenses some margins 

## How to test?
- Pull down branch
- Run `npm start`
- You will see the loading page right away , from there you can go from page to page and see the new styling 
- Also look for a Twitter icon on the `Tweet` page

## Issues:
- This should close #42 
- The board has been updated 

## Screenshots:
![Screen Shot 2020-02-25 at 6 51 08 PM](https://user-images.githubusercontent.com/48968224/75304116-ca44b980-57ff-11ea-9d9f-0fa49efc8599.png)
![Screen Shot 2020-02-25 at 6 51 34 PM](https://user-images.githubusercontent.com/48968224/75304140-d92b6c00-57ff-11ea-94ec-c051525cd621.png)

## Notes:
- Let me know if you want to change anything to this point. I think the bold makes things easier to read. Happy to re think any design stuff. 

Thanks!
@cammac60
